### ISSUE TITLE
X-reader should support reading PDF/UA (PDF/Universal Accessibility) documents and slides

### ISSUE THREAD

**Issue by goebbe** [linuxmint/xreader#571]
```
 * Xreader version (xreader --version)
 * Distribution - (Mint 21)
```

**Issue**
Currently, Xreader does not support pdf/ua standard that has been developed for accessibility.
A pdf reader should enable e.g. blind people to navigate through a pdf/ua document via standard keyboard bindings (e.g. by supporting Orca screen reader).
This includes screen reader support (orca) that support usage of the UI of Xreader AND reading the content of pdf/ua files, structural navigation, navigation by tags, showing/ reading invisible content (like image descriptions) if required. 

Major office applications (LibreOffice/ MS-Office) are able to generate, verify and export pdf/ua documents or slides. 
Xreader should have full support for accessibility features. 

**Steps to reproduce**
Open pdf/ua file with Xreader. 
Enable orca (screen) reader via Preferences > Accessibility > Visual Aids > Screen reader
UI-elements of X-reader are supported by the screen reader with mouse over. 
I could not find a way to get text to speech for the contents of the pdf-files. 

**Expected behaviour**
Text to speech for pdf-content, structural navigation, tables, navigation by tags and access to invisible (pdf/ua) content should be supported out of the box. 

**Other information**
pdf/ua is an ISO and ANSI standard: [https://en.wikipedia.org/wiki/PDF/UA](https://en.wikipedia.org/wiki/PDF/UA)

In EU-European countries, public bodies/ schools/ Universities are legally required to publish all electronic documents in pdf/ua standard format.
It seems that currently there is no free pdf-reader on linux that provides pdf/ua support. :-/

**Comment by goebbe** [comment]
Poppler seems to know about the standard: [https://poppler.freedesktop.org/api/glib/PopplerDocument.html](https://poppler.freedesktop.org/api/glib/PopplerDocument.html): 

PDF Subtype
  POPPLER_PDF_SUBTYPE_PDF_UA 
    ISO 14289 - Document management applications -- Electronic document file format enhancement for accessibility (PDF/UA)

Not sure if the standard is fully supported though.
